public/app/features/variables/pickers/OptionsPicker/OptionsPicker.tsx

```diff
@@ -125,7 +125,7 @@ export const optionPickerFactory = <Model extends VariableWithOptions | Variable
 
       return (
         <VariableLink
-          id={variable.id}
+          id={`var-${variable.id}`}
```

Comment:
maybe consider https://react-spectrum.adobe.com/react-aria/useId.html ?

(and when we upgrade react, https://reactjs.org/docs/hooks-reference.html#useid)

Reply:
Class component, unfortunately :(
Something to consider if/when this gets migrated to a functional component though